gui/src/2D/controls/control.ts

```diff
@@ -230,6 +230,10 @@ export class Control {
         return this._getTypeName();
     }
 
+    /**
+    * An event triggered when pointer wheel is scrolled
+    */
+    public onSVWheelObservable = new Observable<Vector2>();
```

Comment:
SV ?

Reply:
sv = scrollviewer, but nvm...changed to onWheelObservable.